src/main/java/com/synopsys/integration/detect/configuration/DetectProperties.java

```diff
@@ -963,13 +963,20 @@ private DetectProperties() {
             .build();
 
     public static final BooleanProperty DETECT_COMPONENT_LOCATION_ANALYSIS_ENABLED =
-            BooleanProperty.newBuilder("detect.component.location.analysis.enabled", false)
-                    .setInfo("Component Location Analysis Enabled", DetectPropertyFromVersion.VERSION_8_11_0)
-                    .setHelp(
-                            "If set to true, Detect will save an output file named 'components-with-locations.json' in the Scan subdirectory detailing where in the project's source code OSS components are declared.",
-                            "All components will be included when using Synopsys Detect in offline mode. Only policy violating components will be included for Rapid and Stateless Scan modes.")
-                    .setGroups(DetectGroup.GENERAL)
-                    .build();
+        BooleanProperty.newBuilder("detect.component.location.analysis.enabled", false)
+            .setInfo("Component Location Analysis Enabled", DetectPropertyFromVersion.VERSION_8_11_0)
+            .setHelp(
+                    "If set to true, Detect will save an output file named 'components-with-locations.json' in the Scan subdirectory detailing where in the project's source code OSS components are declared.",
+                    "All components will be included when using Synopsys Detect in offline mode. Only policy violating components will be included for Rapid and Stateless Scan modes.")
+            .setGroups(DetectGroup.GENERAL)
+            .build();
+
+    public static final BooleanProperty DETECT_COMPONENT_LOCATION_ANALYSIS_AFFECTS_STATUS =
+        BooleanProperty.newBuilder("detect.component.location.analysis.affects.status", false)
```

Comment:
Question for reviewers: any alternative suggestions for the name of this property or is this OK?

Reply:
Should the property be named as DETECT_COMPONENT_LOCATION_ANALYSIS_STATUS?

Reply:
Seems reasonable to me. I usually like to see if Chris has thoughts here, I added him to the MR.

Reply:
That property sure is a mouthful! lol
I think we are safe if we shrink it down to "detect.component.location.analysis.status" for a tiny bit of brevity, as Dev mentioned.

Reply:
Updated. Thanks for the feedback, everyone!